File: assignment_9_3.py
# buff, pitt, detr

import pandas as pd
from scipy.stats import f_oneway

data = pd.read_excel(r"C:\Users\admin\OneDrive\Desktop\VIT\Sem 4\Foundataion of Data Science\R and Python Codes\data3.xlsx")
a = data["buff"]
b = data["pitt"]
c = data["detr"]

print(f_oneway(a, b, c))

File: assignment9_3.R
# buff, pitt, detr


library("xlsx")

# Read excel file
df <- read.xlsx("C:/Users/admin/Downloads/data33.xlsx", sheetName = "Sheet1")

One_way <- aov(Values~Group, data = df)

cat("One-way ANOVA\n")
print(summary(One_way))